e2e_test_files/twin_test_tempSensor.json

```diff
@@ -0,0 +1,11 @@
+{
+    "moduleId" : "tempSensor",
+    "twinTest": {
```

Comment:
Why not make desired and expectedReported top level entities to avoid TwinTest.TwinTest?

Reply:
Good idea. Doing it.